File: app/src/main/java/com/ladsoft/bakingapp/fragment/di/RecipesFragmentBuilder.java
package com.ladsoft.bakingapp.fragment.di;

import com.ladsoft.bakingapp.fragment.RecipesFragment;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class RecipesFragmentBuilder {
    @ContributesAndroidInjector(modules = RecipesFragmentModule.class)
    abstract RecipesFragment bindFragment();
}

File: app/src/main/java/com/ladsoft/bakingapp/fragment/di/RecipesFragmentBuilder.kt
package com.ladsoft.bakingapp.fragment.di

import com.ladsoft.bakingapp.fragment.RecipesFragment
import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class RecipesFragmentBuilder {
    @ContributesAndroidInjector(modules = [RecipesFragmentModule::class])
    abstract fun bindFragment(): RecipesFragment
}